pinot-controller/src/test/java/com/linkedin/pinot/controller/api/restlet/resources/PinotTableRestletResourceTest.java

```diff
@@ -207,6 +209,17 @@ public void testUpdateTableConfig()
     sendPutRequest(controllerUrlBuilder.forUpdateTableConfig(tableName), tableConfig.toJSON().toString());
     modifiedConfig = getTableConfig(tableName, "REALTIME");
     Assert.assertEquals(modifiedConfig.getQuotaConfig().getStorage(), "10G");
+    try {
+      // table does not exist
+      JSONObject jsonConfig = tableConfig.toJSON();
+      jsonConfig.put("tableName", "noSuchTable_REALTIME");
+      String responseStr =
+          sendPutRequest(controllerUrlBuilder.forUpdateTableConfig("noSuchTable"), jsonConfig.toString());
+      System.out.println(responseStr);
```

Comment:
we should compare this with some error msg? Or, at least validate that this is an error (as opposed to success)?